trade.lic

```diff
@@ -27,7 +27,11 @@ class Trade
       [
         { name: 'duration', regex: /duration/i, optional: false, description: 'Run for a certain amount of time plus the time required to closeup.' },
         { name: 'minutes', regex: /\d+/i, optional: false, variable: true, description: 'Duration of the run in minutes.' },
+        { name: 'hunt', regex: /hunt/i, optional: true, description: 'If Trading xp is above half, hunt occasionally using hunting_info: defined in <name>-caravanhunt.yaml' },
         { name: 'exit', regex: /exit/i, optional: true, description: 'Exits the game after delivering all crates and returning the caravan' }
+      ],
+      [
+        { name: 'hunt', regex: /hunt/i, optional: false, description: 'If Trading xp is above half, hunt occasionally using hunting_info: defined in <name>-caravanhunt.yaml' }
```

Comment:
Does this argument set work the same as the `hunt` above, but without the exit? The exit is an optional setting, so it seems like it would function the same? Sorry, I'm just trying to understand why there's two of the same argument sets, but with a caveat of a single optional exit. Not that this isn't right, I just simply don't follow. :D

Reply:
For the third argset, <duration> and <minutes> are not optional.  So, functionally, your options to use it are:

;trade no args : just runs caravans forever until it's killed.
;trade closeup (exit optional) : runs existing contracts without accepting new ones then ends, quitting if <exit>
;trade hunt : runs caravans forever, but also does hunts occasionally.
;trade duration xminutes <exit> <hunt>: runs trades for Xminutes before closing up and maybe exiting the game.  If hunt is specified it runs hunts during that time period.

So, for example, last night I wanted to test performance in Zoluren, so I set up a hunt file then ran ;trade duration 180 hunt exit.  Which means "Run for 3 hours, hunting occasionally whenever you stop at your hometown, and after 3 hours have passed stop accepting new contracts and exit the game when you're done."